lib/layouts/SubHeader.js

```diff
@@ -0,0 +1,230 @@
+import React, { useState, useRef, useEffect } from "react";
+import * as R from "ramda";
+import classnames from "classnames";
+import { Checkbox, Tab, Button, Dropdown, Input, Pagination } from "lib";
+
+export const SORT_ORDER = [
+  { label: "Ascending", value: "asc" },
+  { label: "Descending", value: "desc" }
+];
+
+const SubHeader = ({
+  searchProps,
+  columnFilterProps,
+  tabsProps,
+  deleteButtonProps,
+  paginationProps,
+  sortProps,
+  changeStateProps,
+  className,
+  toggleFilter,
+  actionBlock
+}) => {
+  const searchRef = useRef();
+  const [searchVisibility, setSearchVisibility] = useState(
+    searchProps ? !searchProps.clear : false
+  );
+
+  const searchParams = new URLSearchParams(location.search);
+
+  const [sortBy, setSortBy] = useState({
+    column:
+      searchParams.get("sort") ||
+      (sortProps && sortProps.options[0] && sortProps.options[0].value),
+    direction: searchParams.get("order") || SORT_ORDER[1].value
+  });
+
+  useEffect(() => {
+    sortProps && sortProps.onClick(sortBy);
+  }, [sortBy]);
+
+  const handleClick = (column, direction) => {
+    setSortBy({
+      ...sortBy,
+      column,
+      direction
+    });
+  };
+
+  const handleSearchVisibilty = status => {
+    setSearchVisibility(status);
+    if (status) {
+      searchRef.current.focus();
+    } else {
+      searchProps && searchProps.clear();
+    }
+  };
+
+  const currentSort =
+    sortProps &&
+    sortProps.options.find(item => item.value === sortBy.column).label;
+
+  const showColumnFilters = R.all(R.either(R.isNil, R.isEmpty), [
+    columnFilterProps,
+    columnFilterProps?.columnFilters,
+    columnFilterProps?.handleColumnFiltering
+  ]);
+
+  const ColumnFiltersDropdown = () => {
+    return (
+      <Dropdown
+        position="bottom-right"
+        label="Columns"
+        buttonStyle="secondary"
+        closeOnSelect={false}
+      >
+        {Object.entries(columnFilterProps.columnFilters).map(
+          ([key, column]) =>
+          (
+            <li
+              key={key}
+              onClick={() =>
+                columnFilterProps.handleColumnFiltering({
+                  name: key,
+                  checked: !column.show
+                })
+              }
+            >
+              <Checkbox
+                onClick={() => {}}
```

Comment:
@vinay0x @edwinbbu @yedhink We have used both onClick & onChange for Checkbox in NeetoHelp. It should be consistent right? Please share your thoughts on which we should use.

References:
https://stackoverflow.com/questions/5575338/what-the-difference-between-click-and-change-on-a-checkbox
https://stackoverflow.com/questions/26615779/react-checkbox-not-sending-onchange

Reply:
@goutham-subramanyam I would suggest to go with `onChange`.  

Ref
https://stackoverflow.com/questions/45617722/react-controlled-checkbox-toggle-on-props

Reply:
Let's use `onChange`. We'll have uniform prop naming.

Reply:
Will make the changes.